common/annotations/dsb.xml

```diff
@@ -14,3345 +14,3345 @@ terms of use, see http://www.unicode.org/copyright.html
 		<annotation cp="🙇" type="tts">se pókłonjeca wósoba</annotation>
-		<annotation cp="🙇‍♂">pónižnosć I muski I se pókłonjecy muski i se pókłoniś</annotation>
+		<annotation cp="🙇‍♂">pónižnosć I muski I se pókłonjecy muski i se pókłoniś | se pókłonjecy muski</annotation>
 		<annotation cp="🙇‍♂" type="tts">se pókłonjecy muski</annotation>
-		<annotation cp="🙇‍♀">pónižnosć I muski I se pókłonjecy muski i se pókłoniś</annotation>
+		<annotation cp="🙇‍♀">pónižnosć I muski I se pókłonjecy muski i se pókłoniś | se pókłonjeca žeńska</annotation>
 		<annotation cp="🙇‍♀" type="tts">se pókłonjeca žeńska</annotation>
-		<annotation cp="🤦">frustrěrowany i nerwowany I woblico I se za głowu pśimjeca wósoba</annotation>
+		<annotation cp="🤦">frustrěrowany i nerwowany I woblico I se za głowu pśimjeca wósoba | se za głowu pśimjeca wósoba</annotation>
 		<annotation cp="🤦" type="tts">se za głowu pśimjeca wósoba</annotation>
-		<annotation cp="🤦‍♂">frustrěrowany I nerwowany I muski I se za głowu pśimjecy muski</annotation>
+		<annotation cp="🤦‍♂">frustrěrowany I nerwowany I muski I se za głowu pśimjecy muski | se za głowu pśimjecy muski</annotation>
 		<annotation cp="🤦‍♂" type="tts">se za głowu pśimjecy muski</annotation>
-		<annotation cp="🤦‍♀">žeńska I frustrěrowany I nerwowany I se za głowu pśimjeca žeńska</annotation>
+		<annotation cp="🤦‍♀">se za głowu pśimjeca žeńska | žeńska I frustrěrowany I nerwowany I se za głowu pśimjeca žeńska</annotation>
 		<annotation cp="🤦‍♀" type="tts">se za głowu pśimjeca žeńska</annotation>
-		<annotation cp="🤷">wšojadno I wšojadno I žedno zdaśe I z rajenjoma sukotajuca wósoba I cwiblowanje</annotation>
+		<annotation cp="🤷">wšojadno I wšojadno I žedno zdaśe I z rajenjoma sukotajuca wósoba I cwiblowanje | z ramjenjoma sukotajuca wósoba</annotation>
 		<annotation cp="🤷" type="tts">z ramjenjoma sukotajuca wósoba</annotation>
-		<annotation cp="🤷‍♂">wšojadno I žedno zdaśe I muski I z rajenjoma sukotajucy muski I cwiblowanje</annotation>
+		<annotation cp="🤷‍♂">wšojadno I žedno zdaśe I muski I z rajenjoma sukotajucy muski I cwiblowanje | z ramjenjoma sukotajucy muski</annotation>
 		<annotation cp="🤷‍♂" type="tts">z ramjenjoma sukotajucy muski</annotation>
-		<annotation cp="🤷‍♀">žeńska I wšojadno I žedno zdaśe I z ramjenjoma sukotajuca žeńska I cwiblowanje</annotation>
+		<annotation cp="🤷‍♀">z ramjenjoma sukotajuca žeńska | žeńska I wšojadno I žedno zdaśe I z ramjenjoma sukotajuca žeńska I cwiblowanje</annotation>
 		<annotation cp="🤷‍♀" type="tts">z ramjenjoma sukotajuca žeńska</annotation>
-		<annotation cp="🧑‍⚕">gójc I gójc/gójcowka I strowotnistwo I chórobny sotša I terapeut</annotation>
+		<annotation cp="🧑‍⚕">gójc I gójc/gójcowka I strowotnistwo I chórobny sotša I terapeut | gójc/gójcowka</annotation>
 		<annotation cp="🧑‍⚕" type="tts">gójc/gójcowka</annotation>
-		<annotation cp="👨‍⚕">gójc I gójcojski kitel I doktor</annotation>
+		<annotation cp="👨‍⚕"> | gójc I gójcojski kitel I doktor</annotation>
```

Comment:
Uhoh………… looks like `I` is supposed to be `|` 🤦‍♂️ Paging CLDR-14898 / #1362

Reply:
@pedberg-icu @macchiati

Reply:
I can't parse "Uhoh………… looks like I is supposed to be"

Reply:
@macchiati Looks like they may be using `U+0049 LATIN UPPERCASE I ` (styled “I” in sans-serif) instead of vertical pipe `|`  similar to in  CLDR-14898 / #1362 where we had dandas instead of vertical pipe. 

The rest was an attempt at humor. "Bring back the danda replacement machinery…"

Reply:
@macchiati New issue filed: "problems in `dsb.xml` " https://unicode-org.atlassian.net/browse/CLDR-14985